requirements.txt

```diff
@@ -0,0 +1,10 @@
+beautifulsoup4==4.5.1
+decorator==4.0.10
+kitchen==1.2.4
+mailcap-fix==0.1.3
+praw==3.5.0
+requests==2.11.0
+rtv==1.11.0
+six==1.10.0
+tornado==4.4.1
```

Comment:
You can get rid of tornado, that was recently removed as a dependency here #278. Other than that, looks good!